openmdao/test_suite/components/exec_comp_for_test.py

```diff
@@ -54,6 +54,10 @@ def __init__(self, exprs, nl_delay=0.01, lin_delay=0.01,
         self.fails = fails
         self.fail_hard = fail_hard
 
+    def setup(self):
```

Comment:
shouldn't need to define setup since the base class already declares the partials.

Reply:
fixed